server.js

```diff
@@ -0,0 +1,4 @@
+const express = require("express")
+
+const app = express();
```

Comment:
```suggestion
const Express = require("express")

const app = Express();
```
classes need to be uppercased